### ISSUE TITLE
Issue with transform-runtime and defualt export using computed property names

### ISSUE THREAD

**Issue by sqal** [babel/babel#7114]
Choose one: is this a bug report or feature request?

### Input Code

```js
// actions.js
const types = {
  FOO: 'FOO',
  BAR: 'BAR',
}

export default {
  [types.FOO]() {},
  [types.BAR]() {},
}

// index.js
import actions from './actions'
```

### Babel/Babylon Configuration (.babelrc, package.json, cli command)

```json
// package.json
"devDependencies": {
  "@babel/core": "^7.0.0-beta.36",
  "@babel/plugin-transform-runtime": "^7.0.0-beta.36",
  "@babel/preset-env": "^7.0.0-beta.36",
  "@babel/runtime": "^7.0.0-beta.36",
  "babel-loader": "^8.0.0-beta.0",
  "webpack": "^3.10.0"
}

// .babelrc
{
  "presets": [
    ["@babel/env", {
      "modules": false,
      "useBuiltIns": "usage"
    }]
  ],
  "plugins": [
    "@babel/transform-runtime"
  ]
}

```

You can clone the repo with reproduction: https://github.com/sqal/babel-7.0.0-beta.36-bug

### Expected Behavior
I expect default import in index.js (actions) to be present

### Current Behavior
Default import in index.js is not present. I am getting an error instead:

` "export 'default' (imported as 'actions') was not found in './actions'`

This is somehow related to transform-runtime plugin along with using computed property names in default export. When I disable transform-runtime, everything works properly. My current workaround for this:

```js

const actions = { ... }

export default actions;
```

### Your Environment

| software         | version(s)
| ---------------- | -------
| Babel            |  7.0.0-beta.36
| node             | 8.5.0
| npm              | 5.5.1
| Operating System | Windows 10

**Comment by nicolo-ribaudo** [comment]
This issue can also be reproduced using only preset-es2015 end exporting an object with more than one compited property: https://babeljs.io/repl/build/master#?babili=false&browsers=&build=&builtIns=false&code_lz=KYDwDg9gTgLgBAE2AMwIYFcA28DeAoOOAbVQF0AuOAIwBoDiBjCxPAXyA&debug=false&circleciRepo=&evaluate=false&lineWrap=false&presets=es2015%2Creact%2Cflow&prettier=false&targets=&version=7.0.0-beta.36

It works with Babel 6.

**Comment by nicolo-ribaudo** [comment]
Fixed in #7116
